src/ngx_http_lua_kong_module.c

```diff
@@ -29,6 +29,14 @@ static int ngx_http_lua_kong_ssl_no_session_cache_flag_index = -1;
 static int
 ngx_http_lua_kong_verify_callback(int ok, X509_STORE_CTX *x509_store);
 #endif
+
+#ifdef NGX_LUA_USE_ASSERT
+#include <assert.h>
+#   define ngx_http_lua_assert(a)  assert(a)
```

Comment:
```suggestion
#   define ngx_http_lua_kong_assert(a)  assert(a)
```

Avoid using the same name as OpenResty's header, otherwise if we `include` it in the future it will cause conflicts.